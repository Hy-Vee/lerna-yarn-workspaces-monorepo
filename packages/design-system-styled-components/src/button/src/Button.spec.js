import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Button from '.';

describe('Button', () => {
    test('primary', () => {
        const tree = renderer.create(<Button>{'Test'}</Button>).toJSON();

        expect(tree).toMatchSnapshot();
    });

    test('secondary', () => {
        const tree = renderer.create(<Button variant="secondary">{'Test'}</Button>).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
