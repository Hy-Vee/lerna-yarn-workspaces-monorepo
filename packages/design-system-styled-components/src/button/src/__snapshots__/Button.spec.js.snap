// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button primary 1`] = `
.c0 {
  background-color: #000;
  border-radius: 4px;
  color: #fff;
  cursor: pointer;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  font-size: 0.88rem;
  font-weight: 500;
  height: 40px;
  line-height: 38px;
  min-width: 200px;
  outline: none;
  overflow: hidden;
  padding: 0 32px;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: capitalize;
  -webkit-transition: all 0.2s ease 0s;
  transition: all 0.2s ease 0s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  white-space: nowrap;
}

.c0:hover {
  color: #000;
  border: 1px solid #000;
  background-color: #fff;
}

<button
  className="c0"
  disabled={false}
>
  Test
</button>
`;

exports[`Button secondary 1`] = `
.c0 {
  background-color: #000;
  border-radius: 4px;
  color: #fff;
  cursor: pointer;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  font-size: 0.88rem;
  font-weight: 500;
  height: 40px;
  line-height: 38px;
  min-width: 200px;
  outline: none;
  overflow: hidden;
  padding: 0 32px;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: capitalize;
  -webkit-transition: all 0.2s ease 0s;
  transition: all 0.2s ease 0s;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  white-space: nowrap;
  color: #666;
  background-color: white;
  border: 1px solid #eaeaea;
}

.c0:hover {
  color: #000;
  border: 1px solid #000;
  background-color: #fff;
}

.c0:hover {
  color: #fff;
  border: 1px solid #000;
  background-color: #000;
}

<button
  className="c0"
  disabled={false}
>
  Test
</button>
`;
